src/pipewire.c

```diff
@@ -166,7 +166,8 @@ ensure_pipewire_is_initialized (void)
 }
```

Comment:
typo in the commit message: certian

It would be good if the commit message mentioned that this is new in pipewire 0.2.6. It is, right ?